gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -425,8 +432,7 @@ nnstreamer_filter_probe (GstTensorFilterFramework * tfsp)
 
   g_return_val_if_fail (nnstreamer_filter_validate (tfsp), FALSE);
 
-  if (checkGstTensorFitlerFrameworkVersion (tfsp->version,
-          GST_TENSOR_FILTER_FRAMEWORK_V0)) {
+  if (GST_TF_FW_V0 (tfsp)) {
```

Comment:
handle !V0 && !V1 case.

Reply:
The successful call to `nnstreamer_filter_validate` ensures that the `tfsp` is either `V0` or `V1`.

Reply:
All the accesses to `priv->fw` happen after the framework has been probed. This means that the filter has been validated. So, this means that the framework version has been checked, and in this case will be limited to V0 and V1.